apps/meteor/app/cloud/server/functions/getWorkspaceAccessTokenWithScope.ts

```diff
@@ -62,6 +66,7 @@ export async function getWorkspaceAccessTokenWithScope(scope = '', throwOnError
 		return {
 			token: payload.access_token,
 			expiresAt,
+			scopes: scope === '' ? [...workspaceScopes] : [scope],
```

Comment:
I wouldn't change this func. Just
```suggestion
			scopes: payload.scope,
```

And keep things simple (same lang as backend, string instead of []string).